@uportal/api-template-vue/src/components/ApiTemplateVue.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="ath-outer">
+    <scale-loader class="ath-spinner" :loading="loading" :height="32" :width="8" :sizeUnit="px"></scale-loader>
+    <div class="ath-inner" style="display: none;">
+      <slot>
+        <h3>No Content Specified</h3>
+        <p>Please specify content for this component using the default slot.</p>
+      </slot>
+    </div>
+  </div>
+</template>
+
+<script>
+import oidc from '@uportal/open-id-connect';
+import { get } from 'axios';
+import Vue from 'vue';
+import { VueSpinners } from '@saeris/vue-spinners';
+
+Vue.use(VueSpinners);
```

Comment:
It may work better to use local component registration https://github.com/Saeris/vue-spinners#-usage

Reply:
This part confused me ...

```
export default {
  components: {
    BarLoader
  },
  // ...
}
```

Would I really need to `export` one of the spinner components?

Reply:
Yes, but with a twist on what you are thinking.
This would not be a new new component, but a property added to `'ApiTemplateVue'`.